ethpm/assets/package.schema.v2.json

```diff
@@ -278,14 +288,15 @@
         "settings": {
           "title": "The settings used for compilation",
           "anyOf": [
-            {"$ref": "solc_Settings"}
+            {"$ref": "#/definitions/solc_Settings"}
           ]
         }
       }
     },
     "solc_Settings": {
```

Comment:
this snake/camel case also makes me uncomfortable - is there a reason it's like this?

Reply:
not that I recall.  I've been thinking of switching us over to `rAndOMCasInG` though.

Reply:
Not horribly opposed to the `raN_DomcAS_ing` - but might cause some heads to explode - and that's no fun for anyone